nixos/modules/services/printing/cupsd.nix

```diff
@@ -334,6 +334,10 @@ in
               [ ! -e "/var/lib/cups/$i" ] && ln -s "${rootdir}/etc/cups/$i" "/var/lib/cups/$i"
             done
 
+            # While cups will automatically create self-signed certificates if accessed via TLS,
+            # this directory to store the certificates needs to be created manually.
+            mkdir -m 0700 -p /var/lib/cups/ssl
```

Comment:
It would make more sense for this to be up at line 314 where the other directories are created

Reply:
Agreed, I wrongly interpreted the lines above for moving `/etc/cups` to `/var/lib/cups` to interfere with `/var[ib/cups/ssl`, but I was wrong. Moved up.